src/lib/sw_engine/tvgSwShape.cpp

```diff
@@ -430,10 +430,26 @@ bool _fastTrack(const SwOutline* outline)
     auto pt3 = outline->pts + 2;
     auto pt4 = outline->pts + 3;
 
-    auto min1 = pt1->y < pt3->y ? pt1 : pt3;
-    auto min2 = pt2->y < pt4->y ? pt2 : pt4;
-    if (min1->y != min2->y) return false;
+    if (*pt1 == *pt4 && *pt2 == *pt3) return false;
```

Comment:
Is this condition necessary?
I think, In tvgSwShape's updateBBox, width and height are already being checked.
I think fastTrack only needs to check if it is rectangle.

Reply:
this condition prevents from filling the whole bbox when the rect has a zero width. 
Since the pictures in the Stress example were so small, the zero width rect occured sometimes and that was the reason why we saw 'the flashes'.

Reply:
I misspelled 'zero width rect' when I meant 'zero width parallelogram' - there are, of course, parallelograms on the pictures in the description.
I also found an issue in my previous solution, so I decided to completely change the algorithm. 
Please have a look now.